source/renderer/app/index.js

```diff
@@ -13,7 +13,7 @@ import actions from './actions';
 import utils from './utils';
 import Action from './actions/lib/Action';
 import translations from './i18n/translations';
-import './themes/index.global.scss';
+import '!style-loader!css-loader!sass-loader!./themes/index.global.scss'; // eslint-disable-line
```

Comment:
@tomothespian This was needed because of importing `react-datetime` stylesheet in `index.global.scss` of theme.
You can test it by removing above section.

Reply:
👍

Reply:
@DominikGuzei does this seem OK to you?

Reply:
What is the specific error you get when removing that part @yakovkaravelov?

Reply:
<img width="445" alt="error" src="https://user-images.githubusercontent.com/1504716/73690524-535c4b00-46af-11ea-93a6-baa2e744a10c.png">

This is the error ☝️I've tried to mess around with Webpack, but no luck. Maybe we should fix this on a different PR.